assets/javascripts/discourse/components/console.hbs

```diff
@@ -0,0 +1,7 @@
+<div
```

Comment:
We should move everything from `/assets/javascripts/discourse` to `/admin/assets/javascripts/discourse`. That will ensure that the JS bundle is only included for staff.